### ISSUE TITLE
Fails to build on ppc64/Linux

### ISSUE THREAD

**Issue by sol** [simonmar/alex#13]
The failure message is:

```
templates/GenericTemplate.hs:77:4:
    Illegal bang-pattern (use -XBangPatterns):
    ! i
```

Tried with ghc 7.6.1 / alex 3.0.2.

**Comment by trofi** [comment]
Likely already fixed by 41e3758b5cc8fc85b7901f59270f6ce550d5ea98

**Comment by andreasabel** [comment]
@simonmar : I suppose this issue can be closed.
